pwiz/data/msdata/SpectrumList_MGF.cpp

```diff
@@ -235,6 +235,16 @@ class SpectrumList_MGFImpl : public SpectrumList_MGF
                                 else
                                     selectedIon.set(MS_selected_ion_m_z, value, MS_m_z);
                             }
+                            else if (name == "INVREION")
+                            {
+                                bal::trim(value);
+                                scan.cvParams.push_back(CVParam(MS_inverse_reduced_ion_mobility, value, MS_volt_second_per_square_centimeter));
+                            }
+                            else if (name == "COLLCROSSSA")
```

Comment:
When moving CCS to the title, it can be parsed back in the `name == TITLE` block above.

Reply:
Done.